power-the-polls-form/src/components/ptp-info-poll-worker/ptp-info-poll-worker.tsx

```diff
@@ -46,19 +46,15 @@ export class PollWorkerInfo {
       const { state, county, city } = this;
       const jurisdictionId = state ? findJurisdictionId( state, county, city ) : null;
 
-      return jurisdictionId != null ?
+      return jurisdictionId != null || state === "MI" ?
          (
             <ptp-info-jurisdiction
-               jurisdictionId={jurisdictionId}
+               jurisdictionId={jurisdictionId || -1}
```

Comment:
why the -1?

Reply:
We know there is a hardcoded check for MI in ptp-info-jurisdiction, so I'd rather pass a number that will never be used vs retyping `jurisdictionId` to allow null and having to account for that in ptp-info-jurisdiction.